python/ml4ir/applications/ranking/tests/test_ranking_model.py

```diff
@@ -76,8 +76,8 @@ def test_csv_and_tfrecord(self):
         )
 
         # Check if the loss and accuracy on the test set is the same
-        assert np.isclose(csv_loss, 0.56748, rtol=0.01)
-        assert np.isclose(csv_mrr, 0.70396, rtol=0.01)
+        assert np.isclose(csv_loss, 0.31154385209083557, rtol=0.01)
```

Comment:
@mohazahran do you know why these might have changed?

Reply:
I expect these changes specially with this type of migration due to changes to how models gets trained under the hood. As long as numbers seems sane, loss if decreasing I believe it should be fine.